projects/api/prisma/seed.ts

```diff
@@ -1,21 +1,49 @@
 import { PrismaClient } from '@prisma/client';
 
+import CreateSupplierDto from '@/suppliers/dto/create-supplier.dto';
+
 import { brands, categories, countries } from './data/products';
 
 const prisma = new PrismaClient();
 
 async function seed() {
-  await prisma.country.createMany({
+  const createdCountries = await prisma.country.createMany({
     data: countries,
   });
 
+  const countryList = await prisma.country.findMany({
```

Comment:
Hmm... please take into account the recommendation about error handling to handle the potential errors that might occur during database operations
Ex:
`try {
  // prima db operations
} catch (error) {
  console.error('Error during blah blah:', error);
  throw error;
} finally {
  await prisma.$disconnect(); 
}`